stable/fairwinds-insights/values.yaml

```diff
@@ -284,6 +284,8 @@ emailCronjob:
     runAsUser: 10324
 
 databaseCleanupCronjob:
+  # -- Enable database cleanup
+  enabled: false
```

Comment:
BTW, do we need to enable this in `ci/test.yaml`?

Reply:
that's a good idea